src/lib/components/navigation/navBar.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+	import { navRoutes } from '../../../routes/utils';
+	import ThemeToggle from '../../components/navigation/themeToggle.svelte';
+	import { writable } from 'svelte/store';
+
+	const open = writable(false);
+
+	const handleClick = () => {
+		open.update(value => !value);
+	};
+</script>
+
+<nav class="nav-container">
+	<div class="nav-logo">
+		<button type="button">
+			<span class="logo">MI_DB</span>
+		</button>
+	</div>
+
+	<div class="nav-items">
+		{#each navRoutes as item}
+			<a href={item.href}>{item.name}</a>
+		{/each}
+	</div>
+
+	<div class="nav-toggle">
+		<button class="nav-toggle-button" on:click={handleClick}>
+			<i class:ri-menu-line={!$open} class:ri-close-line={$open}></i>
+		</button>
+	</div>
+
+	<ThemeToggle />
+
+	<button class="nav-login hidden md:flex w-xl mr-md" type="button">Log in</button>
```

Comment:
also here it is better to have a link element. Just prevent default in case you want to display something on site